Miniscript-Template-000-Timelocks-in-Templates.mediawiki

```diff
@@ -0,0 +1,83 @@
+<pre>
+  Miniscript Template: 0
+  Title: Proposed Timelock Usage
+  Created: 2023-09-11
+</pre>
+
+
+1. Name of Template
+
+Proposed Timelock Usage
+
+2. Goal to be achieved by template
+
+[https://bitcoin.sipa.be/miniscript/ Miniscript] facilitates easier access to utilizing timelocks in Bitcoin, it opens a new frontier in Bitcoin security concerning the construction of more advanced Scripts. This document aims to standardize timelock application across different wallet solutions, focusing on wallet recovery and standardizing expected timelock usage for streamlined hardware and software wallet interfaces.
+
+In the event an output descriptor has been partially or fully lost, minimizing the overall search space for expected timestamp values can expedite recovery. General Miniscript usage supports any valid timelock value, this proposal seeks to guide implementation to more user-friendly practices.
+
+=Block height based Timelocks:=
+
+==Absolute block height based Timelocks==
+
+'''Proposal''': Set absolute block height-based timelocks as multiples of 100, always ending in 00.
+
+Examples of valid Block Height Absolute Timelocks
+
+* after(1000)
+* after(50700)
+* after(82800)
+* after(615000)
```

Comment:
In e2167a422b8504dad625b538d3b3c39d55e39640

All miniscript snippits like this should be in `<code>` blocks. (This is super annoying to type but I still insist on it :)).

Reply:
I'm on it!